userguide/tutorials/overdue.adoc

```diff
@@ -512,6 +512,124 @@ Even if the payment retry schedule is not configured explicitly, there is a defa
 
 If the overdue XML is not configured explicitly, there is a default overdue XML as specified https://github.com/killbill/killbill/blob/29762c013855f7e105861eecd6d5cb393ef98cb8/overdue/src/main/resources/NoOverdueConfig.xml[here]. However, this does not do anything. So, even if a user has unpaid invoices, the user's account will not be overdue.
 
+== More about overdue.xml
+
+Earlier, we have taken a look at a basic overdue configuration file. This section provides more details about the contents of this file.
+
+=== Overdue state Information
+
+At its core, the overdue configuration file contains a list of the various states that a customer's account may go through if the customer does not pay. Each state includes the information listed in the table below.
+
+[cols="a,a,a,a"]
+|===
+|*Information*   |*Type*   |*Description* |*Example*  
+//----------------------
+|name   |Attribute on `state` tag   |Name given to the state | 
+[source, xml]
+<state name="WARNING">  
+|condition     |Tag within `state` |Specifies the condition to be evaluated. The account will enter this overdue state only if the condition evaluates to true.  | See <<Overdue Conditions>>
+|externalMessage   |Tag within `state`    |Optional message which can be retrieved by plugins listening to overdue events, and displayed to the user. Any String value can be specified  |
+[source, xml]
+<externalMessage>
+Reached WARNING
+</externalMessage>
+|subscriptionCancellationPolicy   |Tag within `state`|Specifies whether or not to cancel the subscriptions associated with the account (and the cancellation policy that should be used) when this overdue state is reached. Possible values are `NONE`, `IMMEDIATE`, `END_OF_TERM`)     |
+[source, xml]
+<subscriptionCancellationPolicy>
+END_OF_TERM
+</subscriptionCancellationPolicy>`
+|blockChanges   |Tag within `state`|Specifies whether the customer is allowed to make any plan change on the subscriptions when this overdue state is reached. A true/false value needs to be specified. |
+[source, xml]
+<blockChanges>
+true
+</blockChanges>
+|disableEntitlementAndChangesBlocked  |Tag within `state`|Specifies whether the subscriptions should be paused (which means entitlement and billing will be disabled and changes will be blocked) when this overdue state is reached. A true/false value needs to be specified.    |
+[source, xml]
+<disableEntitlementAndChangesBlocked>
+true
+</disableEntitlementAndChangesBlocked>
+|autoReevaluationInterval   |Tag within `state`   |Specifies the duration after which the overdue state should be reevaluated. The duration needs to be specified with a `unit` (one of `DAYS`, `WEEKS`, `MONTHS`, `YEARS`, `UNLIMITED`) and an integer value. |  [source, xml]
+                <autoReevaluationInterval>
+                 <unit>DAYS</unit>
+                 <number>4</number>
+                </autoReevaluationInterval>
```

Comment:
On the GitHub view of the table, this code is aligned pretty far over to the right. Just want to make sure that's what you wanted.